shared/middle/sicp.md

```diff
@@ -208,8 +208,8 @@ This fundamental work emphasizes controlling the complexity of software systems
 <!-- TODO: add chapter 4-5 questions and fix links to topics. -->
 ## Resources
 
-* [SICP translated into Russian](http://newstar.rinet.ru/~goga/sicp/sicp.pdf)
+* [SICP translated into Russian](https://studizba.com/files/show/pdf/36257-1-h-abel-son-dzh-dzh-sassman-dzh-sassman-.html)
 
 * [Interactive book with examples and exercises executable in browser](http://xuanji.appspot.com/isicp/)
 
-* [SICP learning guide (article in Russian)](https://guides.hexlet.io/how-to-learn-sicp/)
+* [SICP learning guide (article in Russian)](https://guides.hexlet.io/ru/how-to-learn-sicp/)
```

Comment:
Maybe add SICP lectures playlist to Resources  https://youtube.com/playlist?list=PLE18841CABEA24090

Reply:
Done